src/main/java/org/squiddev/plethora/core/PlethoraCore.java

```diff
@@ -67,6 +68,8 @@ public void preInit(FMLPreInitializationEvent event) {
 		IntegrationComputerCraft.setup();
 		IntegrationVanilla.setup();
 		IntegrationForestry.setup();
+		//REFINE Would we benefit from a generalized handler for defining new modules?
```

Comment:
So it'd probably be possible to use `@Mod.EventBusSubscriber` in most of these cases, as we're just registering on the event bus. I'll look into it.

Reply:
Update: I looked into it, and then got side tracked with cleaning out the whole registry code. Just leave it as is for now, and I'll clean it up :).